### ISSUE TITLE
`tag_parts` is not parsed correctly

### ISSUE THREAD

**Issue by scalp42** [y-ken/fluent-mixin-rewrite-tag-name#1]
Hi @y-ken 

Please have a look at: https://github.com/scalp42/fluent-plugin-remote_syslog/blob/fix_RewriteTagName_mixin/lib/fluent/plugin/out_remote_syslog.rb

![](https://i.imgur.com/FNHZJlV.png)

Any chance you could help?

**Comment by scalp42** [comment]
I also tried the following:

```
<match marathon.**>
  type remote_syslog
  host xxx.papertrailapp.com
  port 42000
  tag hostname_command
  hostname example.com
  output_include_time no
  output_include_tag no
  output_data_type attr:container_name,log
</match>
```

But still not being parsed correctly:

![](https://i.imgur.com/MXEJiMe.png)

**Comment by y-ken** [comment]
Hi @scalp42 
Sorry, I kept you waiting. It looks nice.
Does it solved the problem after merged pull-requests of `fluent-plugin-remote_syslog`?
https://github.com/dlackty/fluent-plugin-remote_syslog/pull/3

**Comment by scalp42** [comment]
@y-ken still running into the same issue unfortunately :/

**Comment by dlackty** [comment]
This should have be solved by dlackty/fluent-plugin-remote_syslog#5.
